### DISCUSSION TITLE
Gnuplot install: Failed to download resource "gd_bottle_manifest"

### DISCUSSION THREAD

**Discussion by rwannama** [Homebrew/discussions#1610] [Casks]
"brew install gnuplot" fails with the following error message.

==> Downloading https://ghcr.io/v2/homebrew/core/gd/manifests/2.3.0
##O=#  #                                                                      
curl: (22) The requested URL returned error: 404 
Error: Failed to download resource "gd_bottle_manifest"
Download failed: https://ghcr.io/v2/homebrew/core/gd/manifests/2.3.0

Any advice?

**Comment by carlocab** [comment]
What's the output of `brew config` and `brew doctor`?

**Comment by rwannama** [comment]
Apologies—I fixed some problems revealed by brew doctor and now its working. Thanks for the response.
